src/services/news2/uc/news.facade.js

```diff
@@ -0,0 +1,8 @@
+
+// For inter-component calls
+module.exports = class NewsBcFacade {
+	setup(app, bcContext) {
```

Comment:
Codacy found an issue: ['bcContext' is defined but never used. Allowed unused args must match /app|req|res|next|options|params|^_/u.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=6009087)